cmd/builder/internal/builder/main.go

```diff
@@ -139,7 +169,7 @@ func GetModules(cfg Config) error {
 		return nil
 	}
 
-	if _, err := runGoCommand(cfg, "mod", "tidy", "-compat=1.21"); err != nil {
+	if _, err := runGoCommand(cfg, []string{}, "mod", "tidy", "-compat=1.21"); err != nil {
```

Comment:
You could pass in `nil` instead of `[]string{}`

Reply:
Good catch! I'll update this.